docs/docs/forms.md

```diff
@@ -182,6 +182,62 @@ class FlavorForm extends React.Component {
 
 Overall, this makes it so that `<input type="text">`, `<textarea>`, and `<select>` all work very similarly - they all accept a `value` attribute that you can use to implement a controlled component.
 
+## Handling Multiple Inputs
+
+When you need to handle multiple controlled `input` elements, you can add a `name` attribute to each element and let a handler function choose what to do based on the value of `event.target.name`. For example:
+
+```javascript{14-15,18,32,39}
+class Reservation extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isGoing: false,
+      numberOfGuests: 0
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+  }
+
+  handleInputChange(event) {
+    const pendingState = {};
+    switch (event.target.name) {
```

Comment:
Why don’t we switch on the event target `type` instead?
I would prefer a more succinct version (since that's what the example tries to convey):

```js
handleInputChange(e) {
  const value = e.target.type === 'checkbox' ? e.checked : e.value;
  const name = e.target.name;
  this.setState({
    [name]: value
  });
```